pyproject.toml

```diff
@@ -39,6 +39,7 @@ dependencies = [
     "click~=8.0",
     "click-plugins>=1.0",
     "cligj>=0.5",
+    "importlib-metadata",
     "munch>=2.3.2",
     "setuptools",
```

Comment:
Can setuptools be dropped now, or are there still other cases where setuptools is used during runtime?

Reply:
I'll see and will remove it if possible.